src/module/functions/mod.rs

```diff
@@ -815,14 +817,10 @@ mod tests {
         let types = module.types.add(&[], &[]);
         let (original_fn_id, original_import_id) = module.add_import_func("mod", "dummy", types);
 
-        // Create builder to use inside closure
-        let mut builder = FunctionBuilder::new(&mut module.types, &[], &[]);
-
         // Replace the existing function with a new one with a reversed const value
         let new_fn_id = module
-            .replace_imported_func(original_fn_id, |_| {
-                builder.func_body().unreachable();
-                Ok(builder.local_func(vec![]))
+            .replace_imported_func(original_fn_id, |body| {
+                body.unreachable();
             })
```

Comment:
Remind me what the pattern is to determine the args local ids here? Can we pass them as secondary arguments like `|(body, args)|` kind of thing? Or is there another way?

Reply:
Ah, so I ripped the argument stuff out -- the reason it was in there was so that people who created `FunctionBuilder`s would have the right params/results (to use), but since that now happens [before the builder fn runs](https://github.com/rustwasm/walrus/pull/250/files#diff-3848d3510e4aff5047c97fce1309a78bcdcef84dcc5d5e887d61d51bb8d19af1R487), I don't even have to pass that in anymore.

we *can* pass a pointer to the params & results if we want, though, if you think it is helpful to people who are writing the bodies.

Reply:
Yeah I get that the args and result types aren't available, but what I mean is the `LocalId` references for the args, not the actual information about the args. It might even be nice to have all the info though I think too `|(body, arg_locals, args, results)|`.

Good API design is hard :) But just making sure we don't miss the important `LocalId` info to be able to read an arg without requiring another lookup of the function.

Reply:
My remaining question is just this: if replacing a function that takes an argument, the function localids will remain the same. How do I lookup those function local ids? Do I just assume that the first n locals correspond to the n args? Does that invariant always hold?

Just want to ensure we aren't leaving a gap. Having an example of reading a function argument in the tests would also help.

Reply:
Ahhh I see what you mean, thank you for explaining -- I agree with what having that information would be helpful here... I think it *would* be hard to get it with only the body available.

I'm not sure exactly how you *would* get those function local IDs actually, you'd have to basically remember what the parameters were...

Reply:
Ah I can work with that -- yeah I'll make this change to pass through a pointer to the arg locals. Do you think it wouldn't be useful to pass through the params/results? I know that they're generally already available in some sense but just want to make sure on that